pydantic_evals/pydantic_evals/evaluators/llm_as_a_judge.py

```diff
@@ -62,16 +64,7 @@ async def judge_output(
     If the model is not specified, a default model is used. The default model starts as 'openai:gpt-4o',
     but this can be changed using the `set_default_judge_model` function.
     """
-    user_prompt = dedent(
-        f"""
-        <Output>
-        {_stringify(output)}
-        </Output>
-        <Rubric>
-        {rubric}
-        </Rubric>
-        """
-    )
+    user_prompt = _build_prompt(None, output, rubric)
```

Comment:
Can we use keyword arguments so we can pass `output=output, rubric=rubric` without the ambiguous `None`? Same for the other calls below